test/integration_test.py

```diff
@@ -293,22 +296,41 @@ def __check_manifest(self, file: IO[bytes], uuid_field_name: str):
     def _download_with_drs(self, file_uuid: str):
         base_url = config.service_endpoint() + drs_http_object_path('')
         client = Client(base_url)
-        response = client.get_object(file_uuid)
-        self._validate_fastq_response_content(file_uuid, response.content)
-        log.info('Successfully downloaded file %s with DRS', file_uuid)
+        with self.subTest(access_method=AccessMethod.https):
+            response = client.get_object(file_uuid, access_method=AccessMethod.https)
+            self._validate_fastq_content(file_uuid, response.content)
+            log.info('Successfully downloaded file %s with DRS', file_uuid)
+        with self.subTest(access_method=AccessMethod.gs):
+            gs_url = client.get_object(file_uuid, access_method=AccessMethod.gs)
+            self.assertTrue(gs_url.startswith('gs://'))
+            if 'GOOGLE_APPLICATION_CREDENTIALS' in os.environ:
+                content = self.check_gs_url(gs_url)
+                self._validate_fastq_content(file_uuid, content)
+                log.info('Successfully downloaded from %s with DRS', gs_url)
+            else:
+                log.warning('Cannot verify content of %s because GOOGLE_APPLICATION_CREDENTIALS is not set.', gs_url)
```

Comment:
I think we are starting to embrace the `warnings` framework in Python. 

Can you convert this to a call of https://docs.python.org/3/library/warnings.html#warnings.warn

and monitor PR #1865? It's very close to landing, probably today.

Reply:
I'm not sure exactly what you're asking me to do about #1865

Reply:
I"m asking you to consider the impact that that PR has on your changes or the changes I am requesting. You could rebase on it and test your changes. In the future, when I draw your attention to someone else's PR, in relation to one of yours, that's what I mean.

Note that that PR has now landed.